website/docs/community/resources/oss-projects.md

```diff
@@ -0,0 +1,29 @@
+# OSS Projects
```

Comment:
@gwenwindflower How would you feel about calling this page Open Source Projects instead of OSS Projects?